pipeline/trusted-resources/pkg/trustedtask/trustedtaskrun_test.go

```diff
@@ -116,24 +115,15 @@ func TestVerifyTaskRun_TaskRun(t *testing.T) {
 func TestVerifyTaskRun_OCIBundle(t *testing.T) {
 	ctx := logging.WithLogger(context.Background(), zaptest.NewLogger(t).Sugar())
 
-	// this is needed to enable oci bundle
```

Comment:
May I ask why this configuration is not required any more?

Reply:
it is configured in code already, so we don't need it here. This is a legacy code and not so related to this PR. 
https://github.com/tektoncd/experimental/blob/29ab1898266c8958f62724c45fb36af6224b2425/pipeline/trusted-resources/pkg/trustedtask/trustedtaskrun.go#L98-L100

Reply:
SG